content/en/universal_service_monitoring/_index.md

```diff
@@ -878,3 +80,4 @@ agents:
 [4]: /monitors/types/apm/?tab=apmmetrics
 [5]: /dashboards/
 [6]: /service_management/service_level_objectives/metric/
+[7]: /universal_service_monitoring/setup/
```

Comment:
Some of the links above are probably dead now, but I don't think they're hurting anything